homeassistant/components/media_player/__init__.py

```diff
@@ -491,7 +491,7 @@ class MediaPlayerEntity(Entity):
     _attr_sound_mode: str | None = None
     _attr_source_list: list[str] | None = None
     _attr_source: str | None = None
-    _attr_state: MediaPlayerState | str | None = None
+    _attr_state: MediaPlayerState | None = None
```

Comment:
Sadly, for now we cannot enforce it on the property as there a couple of components which have incorrect implementation.
I think it's still useful to set it on the attribute to ensure new components do not slip through.

Reply:
Move this to separate PR